birdhouse/components/README.rst

```diff
@@ -113,24 +112,27 @@ Comparison between the old and new autodeploy mechanism
 Maximum backward-compatibility has been kept with the old install scripts style:
 
 * Still log to the same existing log files under ``/var/log/PAVICS``.
-* Old single ssh deploy key is still compatible, but the new mechanism allows for different ssh deploy keys for each extra repos (again, public repos should use https clone path to avoid dealing with ssh deploy keys in the first place).
+* Old single ssh deploy key is still compatible, but the new mechanism allows for different ssh deploy keys for each extra
+  repos (again, public repos should use https clone path to avoid dealing with ssh deploy keys in the first place).
 * Old install scripts are kept and can still deploy the old way.
 
 Features missing in old install scripts or how the new mechanism improves on the old install scripts:
 
-* Autodeploy of the autodeploy itself !  This is the biggest win.  Previously, if ``triggerdeploy.sh`` or ``PAVICS-deploy-notebooks`` script changes, they have to be deployed manually.  It's very annoying.  Now they are volume-mount in so are fresh on each run.
-* ``env.local`` now drive absolutely everything, source control that file and we've got a true DevOPS pipeline.
+* Autodeploy of the autodeploy itself !  This is the biggest win.  Previously, if ``triggerdeploy.sh`` (:download:`download <../deployment/triggerdeploy.sh>`)
+  or ``PAVICS-deploy-notebooks`` (TODO: @tlvu Where is this file?) script changes, they have to be deployed manually.  It's very annoying.  Now they are volume-mount in so are fresh on each run.
```

Comment:
@tlvu

Reply:
It's not a file in the source tree, it's the deployed location of the script (/etc/cron.hourly/PAVICS-deploy-notebooks).  Probably change it to ``/etc/cron.hourly/PAVICS-deploy-notebooks`` and leave it without download link.